arcgis-ios-sdk-samples/Content Display Logic/Views/AboutView.swift

```diff
@@ -0,0 +1,145 @@
+// Copyright 2021 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+
+struct AboutView: View {
+    @Environment(\.presentationMode) private var presentationMode
+    
+    var copyrightText: Text {
+        if #available(iOS 15.0, *) {
+            return Text("Copyright © 2015-\(Date(), format: .dateTime.year()) Esri. All Rights Reserved.")
+        } else {
+            let currentYear = Calendar.current.component(.year, from: Date())
+            return Text("Copyright © 2015-\(currentYear) Esri. All Rights Reserved.")
+        }
+    }
+    
+    var body: some View {
+        NavigationView {
+            List {
+                Section {
+                    VStack {
+                        Image("AppIcon2", label: Text("App icon"))
+                        Text(Bundle.main.name)
+                            .font(.headline)
+                        copyrightText
+                            .font(.footnote)
+                            .foregroundColor(.secondary)
+                    }
+                    .listRowBackground(Color.clear)
+                    .frame(maxWidth: .infinity)
+                }
+                Section {
+                    VersionRow(title: "Version", version: "100.13.0.1")
+                    VersionRow(title: "SDK Version", version: Bundle.arcGIS.shortVersion, build: Bundle.arcGIS.version)
+                }
+                Section(header: Text("Powered By")) {
+                    Link("ArcGIS Runtime Toolkit for iOS", destination: .toolkit)
+                    Link("ArcGIS Runtime SDK for iOS", destination: .developers)
+                }
+                Section(footer: Text("Browse and discuss in the Esri Community.")) {
+                    Link("Esri Community", destination: .esriCommunity)
+                }
+                Section(footer: Text("Log an issue in the GitHub repository.")) {
+                    Link("GitHub Repository", destination: .githubRepository)
+                }
+                Section {
+                    Link("API Reference", destination: .apiReference)
+                }
+            }
+            .navigationBarTitle("About", displayMode: .inline)
+            .navigationBarItems(
+                trailing: Button(action: {
+                    presentationMode.wrappedValue.dismiss()
+                }, label: {
+                    Text("Done")
+                        .fontWeight(.semibold)
```

Comment:
I'm not seeing a difference between semibold and bold, so you might consider just using `bold()`.
```suggestion
                        .bold()
```